android/res/xml/provider_paths.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths xmlns:android="http://schemas.android.com/apk/res/android">
+    <external-path name="external_files" path="."/>
+</paths>
```

Comment:
this file is supposed to have a list of file paths our `FileProvider` will use to create URIs.
This is a problem since we let users pick SD Card paths to save after the app has been installed and we can't foresee the future if this information has to be here.

I hope we can hack our way into the `FileProvider`'s paths programmatically when the user changes the default save location.